portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,131 @@
 <!DOCTYPE html>
 <html>
-  <head>
+    <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
-  <body>
+    </head>
+    <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+        <h1>Cynthia HU</h1>
+        <ul class="navbar">
+            <li><a href="#About">About</a></li>
+            <li><a href="#TechCorner">Tech Corner</a></li>
+            <li><a href="#TravelLog">Travel Log</a></li>
+            <li><a href="#Contact">Contact</a></li>
```

Comment:
The naming convention for CSS classes is usually all lowercase delimited by a hyphen. In this case it would be `#tech-corner` https://google.github.io/styleguide/htmlcssguide.html#Capitalization

Reply:
done, changed the ones not following the guide